mmv1/products/dlp/api.yaml

```diff
@@ -1708,3 +1708,151 @@ objects:
                               name: 'radix'
                               description: |
                                 The native way to select the alphabet. Must be in the range \[2, 95\].
+                        - !ruby/object:Api::Type::NestedObject
+                          name: fixedSizeBucketingConfig
+                          description: |
+                            Buckets values based on fixed size ranges. The Bucketing transformation can provide all of this functionality, but requires more configuration. This message is provided as a convenience to the user for simple bucketing strategies.
+
+                            The transformed value will be a hyphenated string of {lowerBound}-{upperBound}. For example, if lowerBound = 10 and upperBound = 20, all values that are within this bucket will be replaced with "10-20".
+
+                            This can be used on data of type: double, long.
+
+                            If the bound Value type differs from the type of data being transformed, we will first attempt converting the type of the data to be transformed to match the type of the bound before comparing.
+
+                            See https://cloud.google.com/dlp/docs/concepts-bucketing to learn more.
+                          properties:
+                            - !ruby/object:Api::Type::NestedObject
+                              name: lowerBound
+                              description: |
+                                Lower bound value of buckets.
```

Comment:
This type of transformation is specific to numeric data:

> This can be used on data of type: double, long.

But, the description shows there's some kind of type conversion that occurs:

> If the bound Value type differs from the type of data being transformed, we will first attempt converting the type of the data to be transformed to match the type of the bound before comparing.

Because of this complexity I think we should leave all the different data types present here ([see Value entity in API docs](https://cloud.google.com/dlp/docs/reference/rest/v2/Value))